.vscode/settings.json

```diff
@@ -51,6 +51,9 @@
     "editor.tabSize": 2,
     "editor.defaultFormatter": "vscode.json-language-features",
   },
+  "[markdown]": {
```

Comment:
We also have this file checked in: https://github.com/eclipse-theia/theia/blob/master/.theia/settings.json.
I’m using the Theia IDE and was asked which Markdown formatter to use. When I selected the DavidAnson one, it was written to .theia/settings.
Should we update both this file and the Theia settings, or check if we can unify them to use only the VS Code one?